backend/src/test/java/com/ambacam/rest/operateurs/requetegroupes/requetes/GroupeRequeteRequetesResourceIT.java

```diff
@@ -31,7 +32,7 @@
 
 import io.restassured.http.ContentType;
 
-public class RequeteGroupeRequetesResourceIT extends ItBase {
+public class GroupeRequeteRequetesResourceIT extends ItBase {
```

Comment:
The name of this class should be `RequeteGroupeRequetesResourceÌT`. Please set it back

Reply:
Done!